packages/overreact/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@microsoft/overreact",
   "description": "TO BE ADDED",
-  "version": "0.1.0-alpha.25",
+  "version": "0.1.0-alpha.26",
```

Comment:
No need to manually change the version number. It will be bumped by `lerna publish`.

@HangtianWu - please create a new PR and revert this part.